src/ODataProvider.cs

```diff
@@ -760,7 +772,8 @@ void HandleResponse(Stream responseStream, HttpStatusCode responseStatusCode, Di
                 {
                     using (var stream = new StreamReader(responseStream))
                     {
-                        result = stream.ReadToEnd();
+                        _endpointResponse = stream.ReadToEnd();
```

Comment:
maybe it is better to make this "_endpointResponse" as an out variable of this method instead of a class variable so it would be easier to understand the code? because now it looks a bit strange

Reply:
sure, I can see that I also is missing not in the if-statements :P

Reply:
ok, better now, can you rename _endpointResponse to endpointResponse? because usually _ means a class variable :)